bokehjs/src/lib/models/tools/edit/edit_tool.ts

```diff
@@ -75,7 +75,8 @@ export abstract class EditToolView extends GestureToolView {
       }
     }
     for (const renderer of renderers) {
-      renderer.data_source.change.emit();
+      renderer.data_source.change.emit()
+      renderer.data_source.properties.data.change.emit()
```

Comment:
without the first, the drag does not update the glyph in a server app, without the second, a data table does not update in a standalone doc

Reply:
Sure, `GlyphRenderer` depends on the former and `DataTable` depends on the later. If you are emitting change signals manually, you have to do this in pairs, to cover all cases.